### PR TITLE
Fix/Disallow duplicate subscription

### PR DESCRIPTION
**Pull request by sasp1**

### CODE REVIEW

File: src/pages/Subscription.tsx [hunk 1]
```diff
@@ -50,8 +51,13 @@ const Subscription = ({ isLoggedIn }: LoggedInProps) => {
       category: "PremiumPage",
       action: "click_gold_page_buy_now",
```
Comment (christianhjelmslund) [inline src/pages/Subscription.tsx:52]: Should be moved into the else statement on line 58 :)

File: src/pages/Subscription.tsx [hunk 2]
```diff
@@ -63,8 +69,13 @@ const Subscription = ({ isLoggedIn }: LoggedInProps) => {
       category: "PremiumPage",
       action: "click_pro_page_buy_now",
     });
```
Comment (christianhjelmslund) [inline src/pages/Subscription.tsx:71]: Same as above

**Review by christianhjelmslund — CHANGES_REQUESTED** [review]

File: src/pages/Subscription.tsx (hunk 1 above)
Comment (sasp1) [reply src/pages/Subscription.tsx]: Should have added a fix now :)

File: src/pages/Subscription.tsx (hunk 2 above)
Comment (sasp1) [reply src/pages/Subscription.tsx]: Should have added a fix now :)

**Review by christianhjelmslund — APPROVED** [review]

**Pull request merged**